Lambdas/GHHooks/EventHandler/Handlers/DocsIssuer.swift

```diff
@@ -65,28 +65,28 @@ struct DocsIssuer {
     }
 
     func getPRsRelatedToCommit() async throws -> [SimplePullRequest] {
-        try await context.githubClient.repos_list_pull_requests_associated_with_commit(
+        try await context.githubClient.reposListPullRequestsAssociatedWithCommit(
             path: .init(
                 owner: self.repo.owner.login,
                 repo: self.repo.name,
-                commit_sha: self.commitSHA
+                commitSha: self.commitSHA
```

Comment:
Now we just need to capitalise this properly 😛

Reply:
I don't think it's possible, automatically, but the generator does take custom key value name mapping pairs and we could use that although I don't care enough for that.